examples/advanced_agents/sandbox_agent/src/nat_sandbox_agent/sandbox/docker_sandbox.py

```
@@ -0,0 +1,286 @@
+                stderr=stderr,
+            )
+
+        except TimeoutError:
+            logger.warning(f"Command timed out after {timeout}s: {command[:50]}...")
+            return CommandResult(
+                exit_code=-1,
+                stdout="",
+                stderr=f"Command timed out after {timeout} seconds",
+            )
+        except ContainerError as e:
+            logger.error(f"Container error: {e}")
```

Ruff (0.14.13)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)